proposals/libocispec.md

```diff
@@ -0,0 +1,37 @@
+# OCI libocispec project proposal
+
+## Abstract
+Provide a C library for parsing and accessing OCI Runtime and Image specification JSON.
+
+## Proposal
+With repositories under the https://github.com/opencontainers organization:
+
+Rename (transfer) https://github.com/giuseppe/libocispec to https://github.com/opencontainers/libocispec.
+
+### Initial Maintainers
+Initial maintainers of the libocispec project:
+* Giuseppe Scrivano <gscrivan@redhat.com> (@giuseppe)
+* Qiang Huang <h.huangqiang@huawei.com> (@hqhq)
```

Comment:
Sorry I'm not available to contribute or maintain this project now, I would recommend my colleague @tanyifeng who's been contributing to and using this project, I think he would be willing and qualified to do this job : )

Reply:
sure.

@tanyifeng, are you fine to be added here?

Reply:
Of course, I'll be glad to be a maintainer of libocispec.